Code/C++/print_shortest_supersequence.cpp

```diff
@@ -0,0 +1,173 @@
+/*
+* Shortest Common Supersequence finds the shortest supersequence which consist of all the characters of both the string in the order specified.
```

Comment:
This program is not linked with the issue #167 
so raise it in another pr

Reply:
@plazzy one file for printing supersequence and  and one file for printing length has been attached.It is related to issue only.
please check it one more time